src/types/pv.ts

```diff
@@ -41,7 +48,8 @@ export class PV {
   public qualifiedName(): string {
     // This can happen if the name is substituted by a macro
     // after the PV object has been created.
-    if (this.name.includes(PV.DELIMITER)) {
+    // Need to make sure that the PV.DELIMITER is not associated with a nested PV.
+    if (this.name.includes(PV.DELIMITER) && !this.name.includes("`")) {
```

Comment:
Very minor point - it looks like you can also create a formula using a single quote (i.e. `'`) as well as with a backtick, e.g. =3+4*'sim://ramp'. This works in Phoebus and PVWS so maybe we could just add an addition `&&` to check that it also doesn't include a `'`?